<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
    background-color: cadetblue;
    border: 2px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
    background-color: cadetblue;
    border: 2px solid black;
}</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>991</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 240, 231);
border: 2px solid black;
}</string>
    </property>
    <property name="text">
     <string>MySQL Connections</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>841</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border: 2px solid rgb(255, 240, 231);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="barcode_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
     </property>
     <property name="text">
      <string>Bar Code</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
}</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="barcode_line">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_line">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>1011</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="memberId_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
       </property>
       <property name="text">
        <string>Member ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="memberId_line">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="firstname_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
       </property>
       <property name="text">
        <string>First name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstname_line">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="surname_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
       </property>
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="surname_line">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="address_button">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
         </property>
         <property name="text">
          <string> Address </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="address_line">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="email_button">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
         </property>
         <property name="text">
          <string>   Email    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="email_line">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pobox_button">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
         </property>
         <property name="text">
          <string>PO Box </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pobox_line">
         <property name="maximumSize">
          <size>
           <width>16777201</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="gender_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gender_combobox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QComboBox:focus{
background-color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="mobile_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>   Mobile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobile_line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="type_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>52</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_combobox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QComboBox:focus{
background-color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Annual Member</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quarterly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Montly</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeView" name="MembersInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>1011</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeView{
background-color: rgb(255, 240, 231);
border : 1px solid black;

 }</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>981</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="addnew_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
       </property>
       <property name="text">
        <string>Add New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showrecord_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
       </property>
       <property name="text">
        <string>Show Record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
       </property>
       <property name="text">
        <string>Save Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="number_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Num in List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_line">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>220</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231);
background-color: blue
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
